services/librariesio/librariesio-sourcerank.service.js

```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+const { colorScale } = require('../../lib/color-formatters')
+const LibrariesIoBase = require('./librariesio-base')
+
+const sourceRankColor = colorScale([10, 15, 20, 25, 30])
+
+class LibrariesIoSourcerank extends LibrariesIoBase {
+  static get category() {
+    return 'other'
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'sourcerank',
+    }
+  }
+
+  static get url() {
+    return this.buildUrl('librariesio/sourcerank')
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Libraries.io SourceRank',
```

Comment:
SourceRank 2.0 is not 'launched', but it is kind of in beta alongside SourceRank 1.0 (at least for some package ecosystems).

e.g:
SourceRank 1.0 breakdown: https://libraries.io/cargo/regex/sourcerank
SourceRank 2.0 breakdown: https://libraries.io/cargo/regex/score

can we clarify that this is a SourceRank 1.0 score here (maybe in the actual badge label too). I'm not sure if it should be in the endpoint. I'm slightly unclear on whether one day SR1 will just disappear and be replaced by SR2 or whether we might at some point need to maintain 2 versions of the badge.

Reply:
Well, we could take the lead from the SourceRank displayed on the libaries.io site. https://libraries.io/cargo/regex has SR 1.0 labeled SourceRank. What if we follow along with that?

It's possible they'll update the `rank` field in this endpoint when they officially launch.

Reply:
Yeah, that's a fair point - given they are still just referring to it as 'SourceRank', we should do the same for now. I'll keep an eye on progress of this anyway